lib/Jsrt/ChakraDebug.h

```diff
@@ -538,7 +538,6 @@
             _In_ unsigned int objectHandle,
             _Out_ JsValueRef *handleObject);
 
-#ifdef _WIN32
```

Comment:
For now just disabling it for nonWindows with a todo note on utf8 implementation.